js/controllers/menu.js

```diff
@@ -17,19 +17,33 @@ function MenuController(tabs) {
   $(document).bind('tabsave', this.onTabSave.bind(this));
 }
 
-MenuController.prototype.onNewTab = function(e, tab) {
-  var id = tab.getId();
-  var name = tab.getName();
-  var listItem = $('<li id="tab' + id + '" draggable="true">' +
-                   '<div class="filename">' + name + '</div>' +
-                   '<div class="close"></div></li>');
-  listItem.bind('dragstart', this.onDragStart_.bind(this, listItem));
-  listItem.bind('dragover', this.onDragOver_.bind(this, listItem));
-  listItem.bind('dragend', this.onDragEnd_.bind(this, listItem));
-  listItem.bind('drop', this.onDrop_.bind(this, listItem));
-  listItem.appendTo($('#tabs-list'));
-  listItem.click(this.tabButtonClicked_.bind(this, id));
-  listItem.find('.close').click(this.closeTabClicked_.bind(this, id));
+MenuController.prototype.addNewTab_ = function(e, tab) {
+  let id = tab.getId();
+  let tabElement = document.createElement("li");
+  tabElement.setAttribute('draggable', 'true');
+  tabElement.id = 'tab' + id;
+  let filenameElement = document.createElement("div");
+  filenameElement.appendChild(document.createTextNode(tab.getName()));
+  filenameElement.className = "filename";
```

Comment:
Single quotes, rather than double quotes please. Here and throughout.

Reply:
Done.